EC2M-20-Controlador-Shared

lauracastilloal97 opened the pull request:
Vista listado de urls con sus respectivos nombres,textos e imágenes

lauracastilloal97 committed:
EC2M-20-Controlador-Shared

Vista listado de urls con sus respectivos nombres,textos e imágenes

boris-burgos commented:
Task linked: [EC2M-20 Hacer controlador del front para las shared urls](https://app.clickup.com/t/2552758/EC2M-20)

boris-burgos reviewed (changes requested):
Las imágenes del storage no deben ir en el git.
Vamos a quitarlas del PR y a añadir en el gitignore que no se añadan.

lauracastilloal97 committed:
Delete d3326be690f05ace3b2e1c96275f1a763a07468c.jpg

lauracastilloal97 committed:
Delete 35b235ab346c54be607b3cd1746c8c6738459272.jpg

lauracastilloal97 committed:
Delete 1eb825babb93a3c1fb69d9a8f47717f2a01620de.png

lauracastilloal97 committed:
Delete 8558141c8c245badb48a70388c5148c0a94a5940.png

lauracastilloal97 committed:
Delete a2b017aea3b5f0620da97e059e3afc6c92e58014.png

lauracastilloal97 committed:
Delete a884b8a2ac26774c5efcec61e14129fd20c51d3a.png

lauracastilloal97 committed:
Archivo ignorar elementos 

Archivo ignorar elementos  dentro de la carpeta public/storage

lauracastilloal97 committed:
Delete d3326be690f05ace3b2e1c96275f1a763a07468c.jpg

lauracastilloal97 committed:
Delete 35b235ab346c54be607b3cd1746c8c6738459272.jpg

lauracastilloal97 committed:
Delete 1eb825babb93a3c1fb69d9a8f47717f2a01620de.png

lauracastilloal97 committed:
Delete 8558141c8c245badb48a70388c5148c0a94a5940.png

lauracastilloal97 committed:
Delete a2b017aea3b5f0620da97e059e3afc6c92e58014.png

lauracastilloal97 committed:
Delete a884b8a2ac26774c5efcec61e14129fd20c51d3a.png

lauracastilloal97 committed:
Delete laravel.log

lauracastilloal97 committed:
Delete 44d142505dd1a3b497197e2f459d2aec779e9ed6

lauracastilloal97 committed:
Delete c24f92750673b2169c8cb7bfbc999cfa8457dc7c

lauracastilloal97 committed:
Update MachineSharedUrl.php

lauracastilloal97 committed:
Update .gitignore

lauracastilloal97 committed:
EC2M-20 Subida archivos controlador

Archivos controlador para mostrar dependiendo de la url, los datos de la machine shared

lauracastilloal97 committed:
Delete listado.blade.php

lauracastilloal97 committed:
Update MachineSharedUrlController.php
